### PR TITLE
added faq page

### PR DESCRIPTION
**Pull request by Dilnazzzshaikh**

### CODE REVIEW

**Review by GauravTheBeginner — DISMISSED** [review]

File: app/faq/page.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import SectionTitle from "@/components/Common/SectionTitle";
+import FAQComponent1 from "../../components/FAQ/SeparetePageFaq";
```
Comment (PrimewiseConsulting) [inline app/faq/page.tsx:3]: rename the page SeparetePageFaq to faqPage

Comment (PrimewiseConsulting) [inline app/faq/page.tsx:3]: what is FAQComponent1 ?

File: app/faq/page.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import SeparetePageFaq from "../../components/FAQ/SeparetePageFaq";
+const faqpage = () => {
+  return (
+    <section id="faq" className="bg-primary/5 py-16 md:py-20 lg:py-28">
+      <div className="container">
+        <SectionTitle title="Frequently Asked Questions" paragraph="" center />
+        <div className="bg-primary/5 py-16 md:py-20 lg:py-1">
+        <SeparetePageFaq/>
+
```
Comment (PrimewiseConsulting) [inline app/faq/page.tsx:12]: why are here extra line spacing given?

File: app/faq/page.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import SectionTitle from "@/components/Common/SectionTitle";
+import FAQComponent1 from "../../components/FAQ/SeparetePageFaq";
+import SeparetePageFaq from "../../components/FAQ/SeparetePageFaq";
+const faqpage = () => {
+  return (
+    <section id="faq" className="bg-primary/5 py-16 md:py-20 lg:py-28">
+      <div className="container">
+        <SectionTitle title="Frequently Asked Questions" paragraph="" center />
```
Comment (PrimewiseConsulting) [inline app/faq/page.tsx:9]: since  paragraph=""  is empty , do we really need it?

File: app/faq/page.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <div className="bg-primary/5 py-16 md:py-20 lg:py-1">
+        <SeparetePageFaq/>
+                  
+          
+        </div>
+        <div></div>
```
Comment (PrimewiseConsulting) [inline app/faq/page.tsx:15]: why do we extra empty div here?

File: app/faq/page.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        
+        <div className="mt-10 ml-0" >
+            <a
+              className="rounded-md bg-primary py-4 px-8 mt-{10rem} text-base font-semibold text-white duration-300 ease-in-out hover:bg-primary/80"
+              href="/contact"
+            >
+              Contact Us
+            </a>
+          </div>
+
```
Comment (PrimewiseConsulting) [inline app/faq/page.tsx:25]: no need of extra line spacing

File: app/faq/page.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+            <a
+              className="rounded-md bg-primary py-4 px-8 mt-{10rem} text-base font-semibold text-white duration-300 ease-in-out hover:bg-primary/80"
+              href="/contact"
+            >
+              Contact Us
+            </a>
+          </div>
+        
+      </div>
+
```
Comment (PrimewiseConsulting) [inline app/faq/page.tsx:27]: same here, this line must be deleted

File: app/page.tsx [hunk 7]
```diff
@@ -6,18 +6,20 @@ import Hero from "@/components/Hero";
...
+import FAQComponent from "@/components/FAQ/FAQComponent";
 
 export default function Home() {
   return (
     <>
       <ScrollUp />
       <Hero />
       <Features />
-      {/* <Video /> */}
+      { <Video /> }
```
Comment (PrimewiseConsulting) [inline app/page.tsx:17]: we dont need {} around Video component , We can remove it and make it consistent as other components.

File: app/page.tsx [hunk 8]
```diff
@@ -6,18 +6,20 @@ import Hero from "@/components/Hero";
...
       <Brands />
       {/* <AboutSectionOne /> */}
       {/* <AboutSectionTwo /> */}
       <Testimonials />
+      <FAQComponent/>
```
Comment (PrimewiseConsulting) [inline app/page.tsx:22]: change  FAQComponent/ to  FAQComponent / to improve consistency

File: components/FAQ/FAQComponent.tsx [hunk 9]
```diff
@@ -0,0 +1,68 @@
+"use client"
+import React, { useState } from 'react';
+
+const FAQComponent = () => {
+  const faqData = [
```
Comment (PrimewiseConsulting) [inline components/FAQ/FAQComponent.tsx:5]: lets make a separate data file for faqData and put the data there, here we should only have the component loading code.

File: components/FAQ/FAQComponent.tsx [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+    },
+    {
+        question: "How can I embark on a journey with Primewise Consulting?",
+        answer:"Getting started with Primewise Consulting is a breeze. Reach out to us through our website or email, and our team will swiftly initiate a conversation to discuss your specific needs and chart out how we can serve you best.",
+    },
+    {
+        question:"Does Primewise Consulting accommodate startups and small businesses?",
+        answer:"Absolutely! Our consultancy services cater to businesses of all sizes, offering custom-tailored solutions to address the distinctive challenges and growth opportunities that each organization presents."
+    }
+
```
Comment (PrimewiseConsulting) [inline components/FAQ/FAQComponent.tsx:30]: why these extra lines here ?

File: components/FAQ/SeparetePageFaq.tsx [hunk 11]
```diff
@@ -0,0 +1,199 @@
+"use client";
+import React, { useState } from "react";
+
+const SeparetePageFaq = () => {
```
Comment (PrimewiseConsulting) [inline components/FAQ/SeparetePageFaq.tsx:4]: rename this page.

File: components/FAQ/SeparetePageFaq.tsx [hunk 12]
```diff
@@ -0,0 +1,199 @@
...
+  const faqData = [
```
Comment (PrimewiseConsulting) [inline components/FAQ/SeparetePageFaq.tsx:5]: again put the data in separate page and load it from there

**Review by PrimewiseConsulting — CHANGES_REQUESTED** [review]

**Review by Dilnazzzshaikh — COMMENTED** [review]
all changes are done

File: app/faq/page.tsx (hunk 1 above)
Comment (Dilnazzzshaikh) [reply app/faq/page.tsx]: renamed the page to faqPage

File: app/faq/page.tsx (hunk 3 above)
Comment (Dilnazzzshaikh) [reply app/faq/page.tsx]: Doesn't need of paragraph so removed it

File: app/faq/page.tsx (hunk 1 above)
Comment (Dilnazzzshaikh) [reply app/faq/page.tsx]: Removed FAQComponant1

File: app/faq/page.tsx (hunk 2 above)
Comment (Dilnazzzshaikh) [reply app/faq/page.tsx]: Extra line spaces are removed

File: app/faq/page.tsx (hunk 4 above)
Comment (Dilnazzzshaikh) [reply app/faq/page.tsx]: Removed empty div tag

File: app/faq/page.tsx (hunk 5 above)
Comment (Dilnazzzshaikh) [reply app/faq/page.tsx]: Removed extra line spacing

File: app/faq/page.tsx (hunk 6 above)
Comment (Dilnazzzshaikh) [reply app/faq/page.tsx]: Deleted extra line space

File: app/page.tsx (hunk 7 above)
Comment (Dilnazzzshaikh) [reply app/page.tsx]: removed {} around Video component

File: app/page.tsx (hunk 8 above)
Comment (Dilnazzzshaikh) [reply app/page.tsx]: changed FAQComponent/ to FAQComponent /

File: components/FAQ/FAQComponent.tsx (hunk 10 above)
Comment (Dilnazzzshaikh) [reply components/FAQ/FAQComponent.tsx]: extra lines are removed

File: components/FAQ/SeparetePageFaq.tsx (hunk 11 above)
Comment (Dilnazzzshaikh) [reply components/FAQ/SeparetePageFaq.tsx]: renamed SeparetePageFaq to faqPage

File: components/FAQ/SeparetePageFaq.tsx (hunk 12 above)
Comment (Dilnazzzshaikh) [reply components/FAQ/SeparetePageFaq.tsx]: created new file to store data,and accessed from there

**Review by Dilnazzzshaikh — COMMENTED** [review]
Changes are done

**Review by GauravTheBeginner — APPROVED** [review]

**Review by PrimewiseConsulting — APPROVED** [review]

**Pull request merged**